### PR TITLE
[SES-258] Delegate Aggregated Views

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket
https://trello.com/c/6zp6dgWU/258-feature-delegatestransparencyreporting-v2

# What solved
✅Should create the scaffolding of the page with the title and the page Story (Empty). 

# Actions
Rename the Delegate page to DelegateReport
Rename index page report and feature flags
Add feature flags
Add scaffolding and stories
Rename hook delegates report
Rename delegate extension

### CODE REVIEW

File: src/core/models/dto/delegatesDTO.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import type { ActivityFeedDto, BudgetStatementDto } from './coreUnitDTO';
 
-export interface DelegatesDto {
+export interface DelegatesReportDto {
```
Comment (YasielCabrera) [inline src/core/models/dto/delegatesDTO.ts:3]: This should keep its name `DelegatesDto` as it is a DTO for the delegates in general, no matter where it is used (like the CoreUnitsDto)

File: src/core/auth/delegatesReportExtension.ts [hunk 2]
```diff
@@ -1,25 +1,25 @@
 import { RoleEnum } from '../enums/roleEnum';
 import type { UserDTO } from '../models/dto/authDTO';
-import type { DelegatesDto } from '../models/dto/delegatesDTO';
+import type { DelegatesReportDto } from '../models/dto/delegatesDTO';
 import type PermissionManager from './permissionManager';
 
-class DelegatesExtension {
+class DelegatesReportExtension {
```
[the remark this answers is not in the document]
Comment (alejandrocabriales) [reply src/core/auth/delegatesReportExtension.ts:6]: ok, I got your to point

File: src/core/businessLogic/builders/delegatesReportBuilder.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import type { ActivityFeedDto, BudgetStatementDto } from '../../models/dto/coreUnitDTO';
-import type { DelegatesDto } from '@ses/core/models/dto/delegatesDTO';
+import type { DelegatesReportDto } from '@ses/core/models/dto/delegatesDTO';
 
-export class RecognizedDelegatesBuilder {
-  private readonly _delegates: DelegatesDto;
+export class RecognizedDelegatesReportBuilder {
```
[the remark this answers is not in the document]
Comment (alejandrocabriales) [reply src/core/businessLogic/builders/delegatesReportBuilder.ts:4]: ok, done

File: src/core/models/dto/delegatesDTO.ts (hunk 1 above)
Comment (alejandrocabriales) [reply src/core/models/dto/delegatesDTO.ts:3]: done

File: src/stories/components/Tabs/Tabs.tsx [hunk 4]
```diff
@@ -173,22 +163,14 @@ const Tabs: React.FC<TabsProps> = ({
     onExpand?.();
   };
 
-  console.log(controlled, selectedTabId);
```
[the remark this answers is not in the document]
Comment (alejandrocabriales) [reply src/stories/components/Tabs/Tabs.tsx:176]: ok,

File: src/stories/components/Tabs/TabsLegacy.tsx [hunk 5]
```diff
@@ -0,0 +1,193 @@
+import styled from '@emotion/styled';
```
[the remark this answers is not in the document]
Comment (alejandrocabriales) [reply src/stories/components/Tabs/TabsLegacy.tsx:1]: ok, done

File: src/stories/components/Tabs/Tab.tsx [hunk 6]
```diff
@@ -6,13 +6,12 @@ import type { WithIsLight } from '@ses/core/utils/typesHelpers';
...
 interface TabProps extends React.PropsWithChildren {
   id?: string;
-  href?: string;
   active: boolean;
   onClick?: (e: React.MouseEvent) => void;
   className?: string;
 }
 
-const Tab: React.FC<TabProps> = ({ children, id, href, active = false, onClick, className }) => {
+const Tab: React.FC<TabProps> = ({ children, id, active = false, onClick, className }) => {
```
[the remark this answers is not in the document]
Comment (alejandrocabriales) [reply src/stories/components/Tabs/Tab.tsx]: done

File: src/stories/containers/RecognizedDelegates/Range.tsx [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+  start: string;
+  end: string;
+}
+
+const Range: React.FC<Props> = ({ start, end }) => (
+  <Container>
+    <ChipRange text={start} />
+    <div
+      style={{
+        // border: '2px solid red',
```
Comment (YasielCabrera) [inline src/stories/containers/RecognizedDelegates/Range.tsx:16]: Did you left this comment here by accident?

File: src/stories/components/Tabs/Tabs.stories.tsx [hunk 8]
```diff
@@ -1,5 +1,5 @@
 import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
-import Tabs from './Tabs';
+import { Tabs } from './TabsLegacy';
```
[the remark this answers is not in the document]
Comment (alejandrocabriales) [reply src/stories/components/Tabs/Tabs.stories.tsx]: done

**Review by YasielCabrera — APPROVED** [review]
LGTM

**Review by YasielCabrera — APPROVED** [review]
LGTM

**Pull request merged**